### PR TITLE
Tv web rebase

### PR DESCRIPTION
**Pull request by jessopb**
This still has a language choosing bug when switching from default (i.e. 'es') to english.
Tested in
yarn dev
yarn dev:web
yarn dev:web-server
yarn run:web-server

need to test desktop build

### CODE REVIEW

**Comment by jessopb** [comment]
Notified @nikooo777  about the lbrytv to web folder rename and that it would affect ansible scripts. This may require coordination at merge?

File: ui/component/header/view.jsx [hunk 1]
```diff
@@ -131,7 +132,7 @@ const Header = (props: Props) => {
         <div className="header__navigation">
           <Button
             className="header__navigation-item header__navigation-item--lbry header__navigation-item--button-mobile"
-            label={__('LBRY')}
+            label={__(LOGO_TITLE)}
```
Comment (seanyesmunt) [inline ui/component/header/view.jsx:135]: This shouldn't be inside `__()`

File: web/stubs/electron.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 export const remote = {
   dialog: {
     showOpenDialog: callable,
   },
   getCurrentWindow: callable,
   app: {
     getAppPath: callable,
-    getLocale: () => 'en',
+    getLocale: () => {
+      console.log('electron stub language');
```
Comment (seanyesmunt) [inline web/stubs/electron.js:13]: console.log

File: web/.env.defaults [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copy this file to .env to make modifications
```
Comment (seanyesmunt) [inline web/.env.defaults:1]: There are two of these? Is that correct?

The other one is up a level

File: config.js [hunk 4]
```diff
@@ -1,13 +1,16 @@
+require('dotenv-defaults').config({silent: false});
```
Comment (seanyesmunt) [inline config.js:1]: I would add a comment here explaining that `.env.defaults` is the base level config

File: custom/homepage.example.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+  // if (showPersonalizedChannels) rowData.push(RECENT_FROM_FOLLOWING);
+  // if (showPersonalizedTags && !showIndividualTags) rowData.push(TRENDING_FOR_TAGS);
+  // if (showPersonalizedTags && showIndividualTags) {
+  //   individualTagDataItems.forEach((item: RowDataItem) => {
+  //     rowData.push(item);
+  //   });
+  // }
+  // rowData.push(TOP_CONTENT_TODAY);
+  rowData.push(LATEST_FROM_LBRYCAST);
```
Comment (kauffj) [inline custom/homepage.example.js:164]: I like this pattern, good call

File: web/.env.defaults (hunk 3 above)
Comment (jessopb) [reply web/.env.defaults:1]: Web packs from inside web folder. /web/.env is read for web.
App packs from root folder.

**Pull request closed without merging**